Automattic/wp-calypso
# Add stylelint-scales

#### Changes proposed in this Pull Request

* Adds a plugin to stylelintrc that allows us to specify font-weights and other style properties to avoid straying from design best practices. See: https://www.npmjs.com/package/@signal-noise/stylelint-scales
* My first time adding something like this so please LMK if there's a process I need to follow or anything I should look out for. :)

#### Testing instructions

- Checkout this branch
- `npm ci`
- Try modifying a sass file and adding a bad font-weight value that isn't 400 or 700
- Try to commit locally
- stylelint should throw errrors on the precommit hook
<img width="1156" alt="Screen Shot 2020-04-10 at 1 45 50 PM" src="https://user-images.githubusercontent.com/1270189/79022187-366e4900-7b32-11ea-989f-3be417aa34e1.png">

- fixing the font-weight violations should allow for a local commit 

<img width="1418" alt="Screen Shot 2020-04-10 at 1 47 53 PM" src="https://user-images.githubusercontent.com/1270189/79022211-471ebf00-7b32-11ea-989b-4c5981541c6b.png">
